CMakePresets.json

```diff
@@ -0,0 +1,60 @@
+{
+    "version": 4,
+    "configurePresets": [
+    {
+      "name": "fprime",
+      "displayName": "F´ Release Preset",
+      "description": "F´ release build using local fprime-venv",
+      "binaryDir": "${sourceDir}/build-fprime-automatic-native",
+      "environment": {
+        "VIRTUAL_ENV": "${fileDir}/fprime-venv",
+        "PATH": "$env{VIRTUAL_ENV}/bin:$penv{PATH}"
+      },
+      "cacheVariables": {
+          "CMAKE_EXPORT_COMPILE_COMMANDS": "ON",
+          "CMAKE_BUILD_TYPE": "Release"
+      }
+    },
+    {
+      "name": "fprime-debug",
+      "inherits": "fprime",
+      "displayName": "F´ Debug Preset",
+      "description": "F´ debug build using local fprime-venv",
+      "cacheVariables": {
+          "CMAKE_BUILD_TYPE": "Debug"
+      }
+    },
+    {
+      "name": "fprime-ut",
+      "inherits": "fprime-debug",
+      "displayName": "F´ Unit Test Preset",
+      "description": "F´ debug build including unit tests using local fprime-venv",
+      "binaryDir": "${sourceDir}/build-fprime-automatic-native-ut",
+      "cacheVariables": {
+          "BUILD_TESTING": "ON"
+      }
+    },
+      {
+        "name": "fprime-ninja",
+        "inherits": "fprime",
+        "displayName": "F´ Release (Ninja) Preset",
+        "description": "F´ release build using ninja and local fprime-venv",
+        "generator": "Ninja"
+      },
+      {
+        "name": "fprime-debug-ninja",
+        "inherits": "fprime-ninja",
```

Comment:
Should this be `"inherits": "fprime-debug",` instead, inheriting everything from UT and adding Ninja generator?